src/styles/primeng-theme/components/_tooltip.scss

```diff
@@ -0,0 +1,12 @@
+.ui-tooltip {
+    &.dot-nav__tooltip {
+        min-width: 100px;
+        max-width: 150px;
+        text-align: center;
+        margin-left: $basic-padding;
+
+        .ui-tooltip-text {
+            padding: $basic-padding $basic-padding * 2;
```

Comment:
Don't we already have this?

Reply:
no, plus we are adding extra padding looks taller than standard.